Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -182,7 +189,7 @@ function getAuthorizationToken(endPoint): Q.Promise<string> {
     return deferred.promise;
 }
 
-async function getAzureRMWebAppID(endpoint, webAppName: string, url: string, headers) {
+async function getAzureRMResourceID(endpoint, resourceName: string, url: string, headers) {
```

Comment:
@arjgupta - Is there anything to react here for the proxy changes that you are doing in parallel?

Reply:
changes to the rest utility will need to be incorporated in my PR. I will do it once these changes are merged. I don't see anything that needs to be change as far as this PR is concerned.